packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -50,7 +49,7 @@ function MainLayoutHeader({
   const isStreakLarge = streak?.current > 99; // if we exceed 100, we need to display it differently in the UI
   const router = useRouter();
   const isSearchPage = !!router.pathname?.startsWith('/search');
-  const { isNewMobileLayout } = useMobileUxExperiment();
+  const isMobileLayout = !useViewSize(ViewSize.Laptop);
```

Comment:
Can't we solve these things with CSS in this case?

Reply:
Yes, many of them actually 🤦‍♀️ thank u

Reply:
I'll go through them 1 by 1 and see what makes sense. Just heads up, since most of mobile layout starts from tablet not just mobile, in some places it may become a mess on css when we try to define something from tablet and below (since at the moment tablet: is used for tablet and above).